docs/src/usage/cookbook.md

```diff
@@ -117,19 +117,98 @@ rs = ADRIA.run_scenarios(dom, p_df, "45")
 
 ```julia
 using ADRIA
-using ADRIA: run_site_selection
-using DataFrames
+using ADRIA: rank_locations
 
 
-dom = ADRIA.load_domain("Example_domain")
+dom = ADRIA.load_domain("path to domain", "45")
+scens = ADRIA.sample_site_selection(dom, 8)
 
-criteria_df = ADRIA.sample_site_selection(dom, 8)  # get scenario dataframe
+# Area of seeded corals in m^2
+area_to_seed = 962.11
 
-area_to_seed = 962.11  # area of seeded corals in m^2
-ts = 5  # time step to perform site selection at
+# Initial coral cover matching number of criteria samples (size = (no. criteria scens, no. of sites)).
+sum_cover = repeat(sum(dom.init_coral_cover; dims=1), size(scens, 1))
+
+# Use rank_locations to get ranks
+ranks = rank_locations(dom, scens, sum_cover, area_to_seed)
+```
 
-# Initial coral cover matching number of criteria samples (size = (no. criteria scens, no. of sites))
-sum_cover = repeat(sum(dom.init_coral_cover, dims=1), size(criteria_df, 1))
+## Intervention location selection - summary functions
 
-ranks = run_site_selection(dom, criteria_df, sum_cover, area_to_seed, ts)
+```julia
+using ADRIA
+using ADRIA:
+    rank_locations,
+    ranks_to_frequencies,
+    location_selection_frequencies,
+    selection_score
+using DataFrames
+using Statistics, StatsBase
+
+# Load data package
+dom = ADRIA.load_domain("path to Domain files", "RCP")
+
+# Select locations for interventions without any model runs
+scens = ADRIA.sample_site_selection(dom, 8)  # Get site selection scenario dataframe.
+
+# Area of seeded corals in m^2
+area_to_seed = 962.11
+
+# Initial coral cover matching number of criteria samples
+sum_cover = repeat(sum(dom.init_coral_cover; dims=1), size(scens, 1))
+
+# Use rank_locations to get ranks
+ranks = rank_locations(dom, scens, sum_cover, area_to_seed)
+
+# Get frequencies with which each site is selected for each rank for set of stand alone
+# location selections
+rank_freq = ranks_to_frequencies(ranks[intervention=1])
+
+# Calculate rank aggregations
+# Get location selection freqencies for set of standalone location selections
+location_selection_frequency = location_selection_frequencies(ranks[intervention=1])
+
+# Get summed inverse rank for set of standalone location selections
+# Measure of magnitude and frequency of high rank
+sel_score = selection_score(ranks[intervention=1])
+
+# Use aggregation function within rank_locations to get direct output
+# To get rank frequencies:
+rank_frequencies_seed = rank_locations(
+    dom, scens, sum_cover, area_to_seed, ranks_to_frequencies, 1
+)
+rank_frequencies_seed = rank_locations(
+    dom, scens, sum_cover, area_to_seed, location_selection_frequencies, 1
+)
+rank_frequencies_seed = rank_locations(
+    dom, scens, sum_cover, area_to_seed, selection_score, 1
+)
+
+# Example using ADRIA runs
+scens = ADRIA.sample(dom, 8)
+rs = ADRIA.run_scenarios(dom, scens, "45")
```

Comment:
I tried to run this here and got the following error, not sure if it is related to this PR though 

```julia
ERROR: BoundsError: attempt to access 75×216×50 NCDatasets.Variable{Float64, 3, NCDatasets.NCDataset{Nothing}} at index [:, :]
Stacktrace:
  [1] interpret_indices_disk(A::NCDatasets.Variable{Float64, 3, NCDatasets.NCDataset{Nothing}}, r::Tuple{Colon, Colon})
    @ DiskArrays C:\Users\pribeiro\.julia\packages\DiskArrays\QlfRF\src\diskarray.jl:114
  [2] getindex_disk(::NCDatasets.Variable{Float64, 3, NCDatasets.NCDataset{Nothing}}, ::Function, ::Vararg{Function})
    @ DiskArrays C:\Users\pribeiro\.julia\packages\DiskArrays\QlfRF\src\diskarray.jl:38
  [3] getindex
    @ C:\Users\pribeiro\.julia\packages\DiskArrays\QlfRF\src\diskarray.jl:211 [inlined]
  [4] getindex(::CommonDataModel.CFVariable{Union{Missing, Float64}, 3, NCDatasets.Variable{Float64, 3, NCDatasets.NCDataset{Nothing}}, NCDatasets.Attributes{NCDatasets.NCDataset{Nothing}}, NamedTuple{(:fillvalue, :missing_values, :scale_factor, :add_offset, :calendar, :time_origin, :time_factor), Tuple{Nothing, Tuple{Float64}, Vararg{Nothing, 5}}}}, ::Colon, ::Colon)
    @ CommonDataModel C:\Users\pribeiro\.julia\packages\CommonDataModel\pO4st\src\cfvariable.jl:390
  [5] load_nc_data(data_fn::String, attr::String, site_data::DataFrame)
    @ ADRIA C:\Users\pribeiro\AIMS\Code\ADRIA.jl\src\io\inputs.jl:127
```